airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/CdcStateCompressionTest.java

```diff
@@ -64,20 +86,32 @@ public void setup() throws Exception {
     // Create a test schema and a bunch of test tables with CDC enabled.
     // Insert one row in each table so that they're not empty.
     testdb.with("CREATE SCHEMA %s;", TEST_SCHEMA);
+    List<Callable<MsSQLTestDatabase>> createAndPopulateTableTasks = new ArrayList<>();
```

Comment:
instead of calling each SQL in a loop, I group them and multithread every step. That allows us to not block on a single table before blocking on the next one, but instead to block on all the slow calls at the same time